airbyte-integrations/connectors/source-facebook-pages/source_facebook_pages/schemas/post.json

```diff
@@ -22,7 +22,7 @@
     },
     "backdated_time": {
       "type": ["string", "null"],
-      "format": "datetime"
+      "format": "date-time"
```

Comment:
Yup- it's date-time! https://docs.airbyte.com/understanding-airbyte/supported-data-types#specific-types